ecs-agent/logger/audit/audit_log.go

```diff
@@ -0,0 +1,24 @@
+// copyright amazon.com inc. or its affiliates. all rights reserved.
```

Comment:
Just something I noticed, how come this copyright blurb doesn't have capitalization?